common/src/service/HttpClientHelper.cs

```diff
@@ -149,15 +122,18 @@ sealed class HttpClientHelper : IHttpClientHelper
                 }
                 else
                 {
-                    await this.ExecuteAsync(
-                       this.httpClientObj,
-                       HttpMethod.Get,
-                       new Uri(this.baseAddress, requestUri),
-                       (requestMsg, token) => AddCustomHeaders(requestMsg, customHeaders),
-                       message => !(message.IsSuccessStatusCode || message.StatusCode == HttpStatusCode.NotFound),
-                       async (message, token) => result = message.StatusCode == HttpStatusCode.NotFound ? (default(T)) : await ReadResponseMessageAsync<T>(message, token).ConfigureAwait(false),
-                       errorMappingOverrides,
-                       cancellationToken).ConfigureAwait(false);
+                    using (var httpClient = buildHttpClient(defaultOperationTimeout))
+                    {
+                        await this.ExecuteAsync(
+                            httpClient,
```

Comment:
there is an extra space before `httpClient,`

Reply:
I'll remove it, good catch